library/cpp/sanitizer/sanitizer/address/dynamic/ya.make

```diff
@@ -0,0 +1,15 @@
+DYNAMIC_LIBRARY()
```

Comment:
looks like the directory `library/cpp/sanitizer/sanitizer` was duplicated by mistake